config/config.go

```diff
@@ -237,6 +237,10 @@ type ConsensusParams struct {
 
 	// max decimal precision for assets
 	MaxAssetDecimals uint32
+
+	// whether to use the old buggy Credential.lowestOutput function
+	// TODO(upgrade): Please remove as soon as the upgrade goes through
+	UseBuggyProposalLowestOutput bool
```

Comment:
Do you think we can name this in a less ...negative.. language ? maybe `ProposalTieBreakerEqualizer`

Reply:
I'm actually a fan of negative language for these sorts of things -- makes us more likely to actually go back and delete the lines afterward! :)
Also having "false" mean "correct behavior" means that test consensus params that don't explicitly set this flag to true (e.g. testBigBlocks) will have correct behavior.